Nodejs/Product/Nodejs/NpmUI/NpmInstallWindowResources.resx

```diff
@@ -192,4 +192,11 @@
   <data name="LatestVersion" xml:space="preserve">
     <value>(latest)</value>
   </data>
+  <data name="OneResultFoundMessage" xml:space="preserve">
+    <value>One result found</value>
```

Comment:
This is pretty nitpicky, but I'm not sure special-casing one localizes well.  In particular, I believe some languages inflect the counted noun differently depending on the quantity (e.g. one, two, many) and would require additional special-cases for strict correctness.  However, I'm probably overthinking things.  If this is the usual pattern, then I'm sure it's fine.

Reply:
I'm not sure if this is the pattern. I would follow up with the loc team and figure out the best approach.